content/en/docs/concepts/configuration/secret.md

```diff
@@ -580,16 +580,35 @@ spec:
   - name: foo
     secret:
       secretName: mysecret
-      defaultMode: 256
+      defaultMode: 0400
 ```
 
 Then, the secret will be mounted on `/etc/foo` and all the files created by the
-secret volume mount will have permission `0400`.
+secret volume mount will have permission `0400`. 
 
 Note that the JSON spec doesn't support octal notation, so use the value 256 for
 0400 permissions. If you use YAML instead of JSON for the Pod, you can use octal
 notation to specify permissions in a more natural way.
 
+Note if you `kubectl exec` into the pod, you need to follow the symlink to find 
+the expected file mode. For example,
+
+```
+$ kubectl exec mypod -it sh
+
+# cd /etc/foo   
+# ls -l
+total 0
+lrwxrwxrwx 1 root root 15 May 18 00:18 password -> ..data/password
+lrwxrwxrwx 1 root root 15 May 18 00:18 username -> ..data/username
+
+# cd /etc/foo/..data
+# ls -l
+total 8
+-r-------- 1 root root 12 May 18 00:18 password
+-r-------- 1 root root  5 May 18 00:18 username
+```
```

Comment:
Kubernetes documentation usually [separates commands from their output.](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output) Please update this to match that style.